### PR TITLE
Add CLA clause to governance document

### LINKED ISSUE #3: CLA required?
It would be good to clarify whether members of the AC, TSC, etc. would need to accept the CLA. I'm assuming they would, but that needs to be explicit (here and on the Google form).

### PR DESCRIPTION
**Pull request by tobie**
Closes #3.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -122,3 +122,11 @@ The Advisory Committee provides perspective and advice to the Technical Steering
...
 
 * The Advisory Committee, TSC and Working Groups may hold calls, video conferences, and face to face meetings. These meetings shall be announced sufficiently in advance with a published agenda. A mechanism by which the community can propose items for the agenda shall be provided.
+
+## Contributor License Agreement
+
+* The AMP Project requires all Owners, Collaborators, and Contributors who open a pull request, to either sign an individual Contributor License Agreement (CLA) or be covered by a corporate CLA in order to protect contributors and users in issues of intellectual property.
+
+* TSC members which aren't already covered by an individual or corporate CLA are required to be covered upon joining the TSC.
+
+* AC members aren't formally required to be covered by a CLA, but will be required to be covered should they decide to contribute to the project (for example by contributing code, documentation, specifications, or design documents) in a way where securing IP commitments is important to keep the projet open source and royalty free.
```
Comment (davidstrauss) [inline GOVERNANCE.md]: > projet

Should be "project".

File: GOVERNANCE.md [hunk 2]
```diff
@@ -122,3 +122,11 @@ The Advisory Committee provides perspective and advice to the Technical Steering
 * A decision may be revisited if new technical information becomes available.
 
 * The Advisory Committee, TSC and Working Groups may hold calls, video conferences, and face to face meetings. These meetings shall be announced sufficiently in advance with a published agenda. A mechanism by which the community can propose items for the agenda shall be provided.
+
+## Contributor License Agreement
+
+* The AMP Project requires all Owners, Collaborators, and Contributors who open a pull request, to either sign an individual Contributor License Agreement (CLA) or be covered by a corporate CLA in order to protect contributors and users in issues of intellectual property.
```
Comment (davidstrauss) [inline GOVERNANCE.md]: > pull request, to either

I don't believe a comma here is necessary.

File: GOVERNANCE.md (hunk 1 above)
Comment (tobie) [reply GOVERNANCE.md]: Duh. Thanks for catching that. Fixed.

File: GOVERNANCE.md (hunk 2 above)
Comment (tobie) [reply GOVERNANCE.md]: Indeed! Fixed.

**Comment by mrjoro** [comment]
I don't see any objections, so merging.  Feel free to revert if the TSC feels otherwise.

**Pull request merged**